### PR TITLE
update files

### PR DESCRIPTION
**Pull request by Abel-desu**
initialize js for popup

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  popup.innerHTML = `<div class="all">
+  <i class="fa-solid fa-xmark close"></i>
+<ul id="popup">
+<li href="#portifolio">portifolio</li>
+<li>about</li>
+<li>contact</li>
+</ul>
+  </div>`;
+  window.appendChild(popup);
+  header.appendChild(window);
```
Comment (omar-muhamad) [inline script.js]: - [ ] Please add the correct style for your mobile menu by positioning every element in the right place like the given design: 
- The close button suppose to be higher at the top and kindly make it bigger and add the correct color to it.
- and the nav links suppose to be lower than that.
So kindly fix that and try to stick as much as possible to the given Figma design.

| Figma Design  | Your Design |
| ------------- | ------------- |
| image  | image  |

File: script.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    header.removeChild(window);
+  });
+  document.querySelector('#portifolio').addEventListener('click', () => {
+    window.removeChild(popup);
+  });
+  document.querySelector('#about').addEventListener('click', () => {
+    window.removeChild(popup);
+  });
+  document.querySelector('#contact').addEventListener('click', () => {
+    window.removeChild(popup);
```
Comment (omar-muhamad) [inline script.js]: - [ ] When I click on any nav link it supposes to redirect me to the right sect on the page and closes the mobile menu that doesn't happen here in your app So please fix that.

**Review by omar-muhamad — CHANGES_REQUESTED** [review]
# Changes Required ♻️
### Hi @Abel-desu,

Nice job, you are almost there.



## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- Github flow is used correctly. ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- [ ] Please check the inline comments below.

## [Optional] suggestions:

- Nothing to mention.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by thecodechaser — APPROVED** [review]
## STATUS: APPROVED 🟢

## Hi @Abel-desu👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



## Optional suggestions
- [ ] Please use camelcase menu options and increase the gap between mobile menu options. It will make your project looks more professional.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**